lib/API.jsx

```diff
@@ -824,6 +824,42 @@ export default function API(network, args) {
                 requireParameters(['cardUserEmail', 'hasCustomLimit', 'domainName'], parameters, commandName);
                 return performPOSTRequest(commandName, parameters);
             },
+
+            /**
+             * Create an admin issued virtual card
+             *
+             * @param {Object} parameters
+             * @param {string} parameters.cardName
+             * @param {string} parameters.assigneeEmail
+             * @param {number} parameters.cardLimit
+             * @param {string} parameters.cardLimitType
+             *
+             * @returns {APIDeferred}
+             */
+            createAdminIssuedVirtualCard: function (parameters) {
+                const commandName = 'Card_CreateAdminIssuedVirtualCard';
+                requireParameters(['cardName', 'assigneeEmail', 'cardLimit', 'cardLimitType'], parameters, commandName);
+                return performPOSTRequest(commandName, parameters);
+            },
+
+            /**
+             * Edit an admin issued virtual card
+             *
+             * @param {Object} parameters
+             * @param {string} parameters.domainName
+             * @param {number} parameters.cardID
+             * @param {string} parameters.cardName
+             * @param {string} parameters.assigneeEmail
+             * @param {number} parameters.cardLimit
+             * @param {string} parameters.cardLimitType
+             *
+             * @returns {APIDeferred}
+             */
+            editAdminIssuedVirtualCard: function (parameters) {
+                const commandName = 'Card_EditAdminIssuedVirtualCard';
+                requireParameters(['domainName', 'cardID', 'cardName', 'assigneeEmail', 'cardLimit', 'cardLimitType'], parameters, commandName);
```

Comment:
```suggestion
                requireParameters(['domainName', 'cardID', 'cardTitle', 'assigneeEmail', 'cardLimit', 'cardLimitType'], parameters, commandName);
```

Maybe we should standardize on using cardTitle instead of cardName for admin-issued virtual cards everywhere 🤔

Reply:
I think it would be a good idea to standardize, yes. I've found one spot where we're using `cardName` instead of `cardTitle` here:
https://github.com/Expensify/Web-Expensify/blob/9cb0451202f9fd77b138b45f2227a2a8d7ba309f/site/app/domainEditor/ConfigureAdminIssuedVirtualCard.jsx#L15-L16